src/layouts/base-layout/index.tsx

```diff
@@ -58,6 +58,34 @@ const BaseLayout = ({
 	useScrollPercentageAnalytics()
 	const [showSkipLink, setShowSkipLink] = useState(false)
 
+	if (process.env.NODE_ENV === 'development') {
+		useEffect(() => {
+			const clientId = crypto.randomUUID()
+			const eventSource = new EventSource(`/api/refresh?id=${clientId}`)
+
+			eventSource.onopen = () => {
+				console.log(`Reload Client ${clientId} Connected`)
+			}
+
+			eventSource.onmessage = (event) => {
+				const data = event.data && JSON.parse(event?.data)
+
+				if (data.reload) {
+					console.log('Reload Client Reloading Page')
+					window.location.reload()
+				}
+			}
```

Comment:
The same, but in a try/catch block.
```suggestion
				try {
				  const data = event.data && JSON.parse(event?.data)
  
				  if (data.reload) {
					  console.log('Reload Client Reloading Page')
					  window.location.reload()
				  }
				} catch (err) {
					console.error('Error parsing message data:', err);
				}
```